### DISCUSSION TITLE
How to use the administrator account /master level account

### DISCUSSION THREAD

**Discussion by zeroesh** [webreinvent/vaahcms#94] [Help]
I want to build multiple websites using the cms so i have few questions
1. how to access the master level account and create new accounts for multiple websites
2. how to add the domain for the site to host the specific clients website

**Comment by themodernpk** [comment]
@zeroesh did you mean that you want to have one instance of `VaahCMS` and host multiple websites with different domains? If yes, will each website be going to have different content?

**Reply by zeroesh** [reply]
yes they will be on different domains and yes different contents, menus, images

**Reply by themodernpk** [reply]
@zeroesh it's a bit complex use case and `VaahCMS` does not support it out of the box but as it is built on Laravel, you can do everything which is possible in Laravel. You can do custom routing in your VaahCMS theme and use the https://github.com/gecche/laravel-multidomain package. There is no straightforward tutorial to present.

There is another approach to building a multi-tenant SaaS application and for that, you can use https://github.com/webreinvent/vaahcms-module-saas, but again no straightforward tutorial.
